utilities/rpp-unittests/OCL/Single_ocl.cpp

```diff
@@ -0,0 +1,533 @@
+        rppi_jitter_u8_pkd3_gpu(d_input, srcSize[0], d_output, kernelSize, handle);
+        break;
+    case 2:
+        test_case_name = "blur";
+        rppi_blur_u8_pkd3_gpu(d_input, srcSize[0], d_output, kernelSize, handle);
+        break;
+    case 3:
+        test_case_name = "brightness";
+        rppi_brightness_u8_pkd3_gpu(d_input, srcSize[0], d_output, alpha, beta, handle);
+        break;
+    case 4:
+        test_case_name = "blend";
+        rppi_blend_u8_pkd3_gpu(d_input, d_input_second, srcSize[0], d_output, alpha, handle);
+        break;
+    case 5:
+        test_case_name = "color_temperature";
+        rppi_color_temperature_u8_pkd3_gpu(d_input, srcSize[0], d_output, adjustmentValue, handle);
+        break;
+    case 6:
+        test_case_name = "gamma_correction";
+        rppi_gamma_correction_u8_pkd3_gpu(d_input, srcSize[0], d_output, gamma, handle);
+        break;
+    case 7:
+        test_case_name = "fog";
+        rppi_fog_u8_pkd3_gpu(d_input, srcSize[0], d_output, fogValue, handle);
+        break;
+    case 8:
+        test_case_name = "snow";
+        rppi_snow_u8_pkd3_gpu(d_input, srcSize[0], d_output, snowPercentage, handle);
+        break;
+    case 9:
+        test_case_name = "lens_correction";
+        rppi_lens_correction_u8_pkd3_gpu(d_input, srcSize[0], d_output, strength, zoom, handle);
+        break;
+    case 10:
+        test_case_name = "noise";
+        rppi_noise_u8_pkd3_gpu(d_input, srcSize[0], d_output, noiseProbability, handle);
+        break;
+    case 11:
+        test_case_name = "pixelate";
+        rppi_pixelate_u8_pkd3_gpu(d_input, srcSize[0], d_output, handle);
+        break;
+    case 12:
+        test_case_name = "exposure";
+        rppi_exposure_u8_pkd3_gpu(d_input, srcSize[0], d_output, exposureFactor, handle);
+        break;
+    case 13:
+        test_case_name = "fisheye";
+        rppi_fisheye_u8_pkd3_gpu(d_input, srcSize[0], d_output, handle);
+        break;
+    case 14:
+        test_case_name = "vignette";
+        rppi_vignette_u8_pkd3_gpu(d_input, srcSize[0], d_output, stdDev, handle);
+        break;
+    case 15:
+        test_case_name = "flip";
+        rppi_flip_u8_pkd3_gpu(d_input, srcSize[0], d_output, flipAxis, handle);
+        break;
+    case 16:
+        test_case_name = "rain";
+        rppi_rain_u8_pkd3_gpu(d_input, srcSize[0], d_output, rainPercentage, rainWidth, rainHeight, transparency, handle);
+        break;
+    case 17:
+        test_case_name = "rotate";
+        rppi_rotate_u8_pkd3_gpu(d_input, srcSize[0], d_output, dstSize[0], angle, handle);
+        break;
+    case 18:
+        test_case_name = "warp-affine";
+        rppi_warp_affine_u8_pkd3_gpu(d_input, srcSize[0], d_output, dstSize[0], affine, handle);
+        break;
+    case 19:
+        test_case_name = "resize";
+        rppi_resize_u8_pkd3_gpu(d_input, srcSize[0], d_output, dstSize[0], handle);
+    case 20:
+        test_case_name = "resize_crop";
```

Comment:
Codacy found an issue: [Variable 'test_case_name' is reassigned a value before the old one has been used. 'break;' missing?](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/rpp/pullRequest?prid=5976402)